packages/components/tests/dummy/app/styles/app.scss

```diff
@@ -19,13 +19,16 @@
 @import "./components/dummy-component-props";
 @import "./components/dummy-placeholder";
 
+[class^='.dummy-'] {
```

Comment:
[issue] can you apply this where you need and not at such high level? we can discuss how to tackle this in a more systematic way (set up a 15/20 mins meeting with me, where discuss the possible options we have).

Reply:
Really it should be `*, *::after, *::before { box-sizing: border-box; }; I think we have an issue in the backlog about it. I was trying just to target the dummy styles for now because it's tedious to have to write this every time. But for now, I will remove it and apply it to all the classes instead.